lib/order.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/menu_module.rb'
+require './lib/text_module.rb'
+
+class Order
+
+  include Text, Menu
+
+  attr_reader :current_order
+
+  def initialize
+    @current_order = []
+    @total = 0
+    super
+  end
+
+  def add_to_order(item, quantity, price)
+    raise "Item not on menu" unless on_menu?(item)
+    raise "Incorrect price, order rejected" unless correct_price?(item, quantity, price)
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)